CHANGELOG.rst

```diff
@@ -5,6 +5,13 @@ Release Notes
 
 Changes are ordered reverse-chronologically.
 
+0.7.1
+-----
+
+ - Improvements and fixes to installers including Windows & Debian
+ - Updated documentation
```

Comment:
I think we're loosing details because of the work-flow. We should add change log entries in PRs instead.

These release notes may not be inaccurate, but they don't state anything specific, so they're hard to use.

Reply:
Would like to suggest that we track a detailed and thorough release note chapter for each release - like this: http://docs.wagtail.io/en/latest/releases/2.0.html

Notice the "IN DEVELOPMENT" in the header, useful until the release is actually final.

After having added lots of release notes, we can always boil it down, edit the order etc.

And we can keep a separate, selective release note "executive version", of course, since there seems to be an idea that this is also useful.